### PR TITLE
Sylius Plus modular installation and upgrading

### PR DESCRIPTION
**Pull request by CoderMaggie**
| Q               | A
|-----------------|-----
| Branch?         | 1.12
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

Since the introduction of the Sylius Plus modular approach, we haven't updated the installation instructions on the official documentation.

### CODE REVIEW

**Review by TheMilek — APPROVED** [review]

File: docs/book/installation/sylius_plus_installation.rst [hunk 1]
```diff
@@ -1,384 +1,28 @@
-
 .. rst-class:: plus-doc
 
 Sylius Plus Installation
 ========================
```
Comment (TheMilek) [inline docs/book/installation/sylius_plus_installation.rst:4]:
```suggestion
Sylius Plus
========================
```
I'm just wondering if this filename and topic are still misleading after these changes and if they should be renamed to something more general

Comment (CoderMaggie) [reply docs/book/installation/sylius_plus_installation.rst:4]: This is not a doc telling you what is Sylius Plus but how to install it :D even though it does not contain specific steps anymore.

File: docs/book/installation/sylius_plus_installation.rst [hunk 2]
```diff
@@ -1,384 +1,28 @@
...
-    bin/console assets:install --ansi
+Sylius Plus is a licensed, modular set of advanced extensions to Sylius applications that add new features and experience.
+As these are private packages they cannot be installed by every Sylius user, but only by those having the license.
 
-**12.** Rebuild cache for proper display of all translations:
+Installing Sylius Plus modules as plugins to a Sylius application
+-----------------------------------------------------------------
 
-.. code-block:: bash
+Sylius Plus consists of modules in a flexible licensing model, where you can choose the features you need. `Read more! <https://sylius.com/plus/?utm_source=docs>`_
```
Comment (mpysiak) [inline docs/book/installation/sylius_plus_installation.rst:12]:
```suggestion
Sylius Plus consists of modules in a flexible licensing model, where you can choose the features you need. `Read more! <https://sylius.com/plus>`_
```

**Review by mpysiak — APPROVED** [review]

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, Magdalena! :1st_place_medal:

**Pull request merged**